src/Inventory/Request.php

```diff
@@ -97,7 +97,6 @@ protected function handleAction($query, $content = null): bool
             case self::DEPLOY_ACTION:
             case self::WOL_ACTION:
             default:
-                $this->addError("Query '$query' is not supported.", 400);
```

Comment:
I do not agree. This is not supported, and GLPI must tell that; the only way to handle those queries is currently to use a plugin; but user won't known there is something wrong if we just "ignore".

Reply:
Indeed here I probably was confused with the handleTask() case below and the agent does not even support these tasks so I just forgot to double-check this case.
I will remove this change.

Reply:
Just reverted this one.